app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -271,10 +274,17 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
             className="flex flex-col items-center w-full pt-8"
             data-testid="weekly-picks"
           >
-            <h1 className="pb-8 text-center text-[2rem] font-bold text-foreground">
-              Week {week} pick
-            </h1>
-
+            <Heading 
+              as='h1'
+              className='pb-8'
+            >{`Week ${week} pick`}
+            </Heading>
+            <Heading
+              as='h2'
+              className='pb-8 text-muted-foreground'
+              data-testid='entry-name'
```

Comment:
Let's follow BEM naming convention. This should be `data-testid='week__entry-name'`

Reply:
@choir241